app/views/dashboard.pug

```diff
@@ -2,20 +2,21 @@ doctype html
 html(lang="en")
     head
         title User Dashboard
-        script (src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js")
         link(rel="stylesheet", href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css")
         link(rel="stylesheet", href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap-theme.min.css")
+        link(rel="stylesheet", src ="/externalCSS.css")
         script(src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js")
+        script(type="text/javascript" src="//ajax.googleapis.com/ajax/libs/jquery/2.0.0/jquery.min.js")
 
-    body(style='width:100%;height:100%;background-image:url("/assets/background.png");')
+    body(id='full' style='background-image:url("/assets/background.png");')
 
-        div.container(style='width:100%;height:100%;')
+        div.container(id='full')
 
-            div(id='topHalf' style='float:top;height:20%;')
+            div(id='topFloat')

```

Comment:
Try having this title not touch the left side of the page so the button bar can span the whole left side.